assets/dev/js/editor/regions/navigator/elements/hooks/ui/panel/editor/open/navigator-toggle-list.js

```diff
@@ -0,0 +1,53 @@
+import Base from '../../../base/base';
+
+/**
+ * Navigator toggle list hook, is responsible for recursive toggling of selected widget for user indication.
+ */
+export class NavigatorToggleList extends Base {
+	getCommand() {
+		return 'panel/editor/open';
+	}
+
+	getId() {
+		return 'navigator-toggle-list';
+	}
+
+	getConditions( args ) {
+		if ( ! super.getConditions() ) {
+			return false;
+		}
+
+		/**
+		 * Since on the first creation of new dragged widget there is no yet 'navigator.view'
+		 * And the navigator is open its required to re-run the hook to apply it on first time.
+		 */
+		if ( ! args.view.container.navigator.view ) {
+			setTimeout( () => $e.hooks.ui.get( this.getId() ).callback( args ) );
+			return false;
+		}
+
+		return true;
+	}
+
+	apply( args ) {
+		this.toggleList( args.view.container.navigator.view );
+	}
+
+	/**
+	 * @param {Element} view
+	 */
+	toggleList( view ) {
+		view.recursiveParentInvoke( 'toggleList', true );
+
+		const { region } = $e.components.get( 'navigator' ),
+			layout = region.getLayout();
+
+		layout.elements.currentView.recursiveChildInvoke( 'removeEditingClass' );
+
+		view.addEditingClass();
+
+		elementor.helpers.scrollToView( view.$el, 400, layout.elements.$el );
```

Comment:
It's now an option of `document/elements/select` command.

Reply:
![image](https://user-images.githubusercontent.com/10234691/142200358-6a5cbb0c-ceb7-4f9d-a2ab-a68d29ef2294.png)
Problematic, I'm suggest to make `scrollToView` internal command.
`$e.internal( 'document/elements/scroll-to-view')`

Reply:
@iNewLegend good idea, can you do it?

Reply:
Handled.